api/routes/web.php

```diff
@@ -17,3 +17,7 @@
 Route::get('/', function () {
   return ['Laravel' => app()->version()];
 });
+
+Route::get('/get_ini_values', function () {
+  return ['upload_max_filesize' => ini_get('upload_max_filesize'), 'max_file_uploads' => ini_get('max_file_uploads')];
+});
```

Comment:
Is it possible to move this in the `api.php` file inside the middleware so that the route can be protected?

Reply:
Thanks for this, updated the routes to `api/ini` inside the `api.php`. The route now is protected.